cross-chain/arbitrum/external/mainnetArbitrum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0x0b2402144Bb366A632D14B83F244D2e0e21bD39c",
```

Comment:
Double-checked with https://book.wormhole.com/reference/contracts.html#token-bridge and confirmed the address is `0x0b2402144Bb366A632D14B83F244D2e0e21bD39c `.